package.json

```diff
@@ -12,6 +12,9 @@
   "workspaces": [
     "packages/*"
   ],
+  "resolutions": {
+    "@stoplight/json-schema-sampler": "0.2.0"
```

Comment:
Why the need for the resolution? Do resolutions make sense in a lib? Are they published? If so, does NPM handle them?

Reply:
Prism has an explicit version set, and that's why.
I decided to explicit version, because that's what npm suggests - theoretically all `0.x.x` can come up with breaking changes, as this particular major version means the API is not stable yet, hence I wanted to make sure Prism is safe.

>  Do resolutions make sense in a lib?

I added them only to make the build passing.
Similarly to Platform, Elements use an older version of Webpack that's not capable of recognizing optional null operator, that caused the build to fail

I fixed that in a newer version -> https://github.com/stoplightio/json-schema-sampler/commit/e096188662e08bd16e9f4982e69cd27f36b1df25 and added a few useful features altogether, i.e. https://github.com/stoplightio/json-schema-sampler/commit/bdd5ae5a6e57289ef2508f8abb3065231d2d88a1

I'll try to update this dependenct in Prism in order to remove that resolution.

> If so, does NPM handle them?

nope, ignored